plugins/flatpak/gs-flatpak.c

```diff
@@ -4133,7 +4133,7 @@ gs_flatpak_update_app (GsFlatpak *self,
 
 		/* either install or update the ref */
 		ref_display = gs_flatpak_app_get_ref_display (app_tmp);
-		if (!is_proxy_app && !g_hash_table_contains (hash_installed, ref_display)) {
```

Comment:
Could it have been that the intention here was to check whether `app_tmp` was a proxy app instead? That would make more sense.

Reply:
I think so, but not because there may be a proxy app related to others (and this is not what was being done).